### PR TITLE
GEDIL4B

### PR DESCRIPTION
**Pull request by sdradsb**
GEDIL4B DOC

### CODE REVIEW

**Review by abarciauskas-bgse — CHANGES_REQUESTED** [review]
Made some comments via reviewnb, thanks @sdradsb

File: docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb [hunk 1]
```diff
@@ -0,0 +1,425 @@
+{
```
Comment (abarciauskas-bgse) [inline docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb:1]: <p>This should be GEDI L4B I believe</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (abarciauskas-bgse) [inline docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #5.    !pip install geojsoncontour
</pre><p>I don't believe any of these are used so this section should be removed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (abarciauskas-bgse) [inline docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #4.    username = maap.profile.account_info()["username"]
</pre><p>Is there a reason to print your username?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (abarciauskas-bgse) [inline docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    COLLECTIONID = 'C1202077968-NASA_MAAP' # specify the collection id for the ATLAS dataset
</pre><p>I would use the collection variable you defined above here to get this id programmatically - i.e. <code>collection_concept_id = colllection['concept-id']</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (abarciauskas-bgse) [inline docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #4.    print(data.shape)
</pre><p>Is there a reason to print the data.shape? I would add a comment why you print the data.shape.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (abarciauskas-bgse) [reply docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb:1]: <p>I would just remove this line altogether</p>

File: docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
```
Comment (abarciauskas-bgse) [inline docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.    COLLECTIONID = collection['concept-id'] # specify the collection id for the ATLAS dataset
</pre><p>This comment should say "specify the collection id for the GEDI dataset"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

File: docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb (hunk 1 above)
Comment (sumantjha) [reply docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb:1]: <p>This has not been fixed yet.</p>

File: docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
```
Comment (sumantjha) [inline docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb:1]: <h2>... GEDIL4B ...</h2><p>should be</p><h2>...GEDI L4B ...</h2>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (sumantjha) [inline docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb:1]: <p>It's a good practice to import all packages together on the top of the code. However, you can ignore it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (sumantjha) [inline docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb:1]: <p>#print the downloaded file</p><p> </p><p>&gt;&gt;</p><pre class="ql-syntax" spellcheck="false"># Print the downloaded file.
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (sumantjha) [inline docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #5.    #printing the shape of data to verify if rasterio was able to read the file properly

&gt;&gt;&gt;

Line #5.    # Printing the shape of data to verify if rasterio was able to read the file properly.
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (sumantjha) [inline docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb:1]: <p>plot the .tif file using 'matplotlib'</p><p> </p><p>&gt;&gt; Plot the .tif file using 'matplotlib'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (sumantjha) [inline docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb:1]: <p>Show the plot output and add that to the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

**Comment by abarciauskas-bgse** [comment]
@sumantjha I have addressed the comments we made - could you re-review?

**Review by sumantjha — APPROVED** [review]
LGTM

**Review by abarciauskas-bgse — APPROVED** [review]

**Pull request merged**